src/site/content/en/blog/local-fonts/index.md

```diff
@@ -174,54 +173,28 @@ The Local Font Access API is an attempt at solving these challenges. It consists
 To check if the Local Font Access API is supported, use:
 
 ```js
-if ('queryLocalFonts' in self) {
+if ('queryLocalFonts' in window) {
   // The Local Font Access API is supported
 }
 ```
 
-{% Aside %}
-The Local Font Access API is currently only available on desktop versions of Chromium, but not on
-mobile operating systems like Android or iOS.
-{% endAside %}
-
-#### Asking for permission
-
-Access to a user's local fonts is gated behind the `"local-fonts"` permission, which you can request
-with
-[`navigator.permissions.request()`](https://w3c.github.io/permissions/#requesting-more-permission).
-
-```js
-// Ask for permission to use the API
-try {
-  const status = await navigator.permissions.request({
-    name: 'local-fonts',
-  });
-  if (status.state !== 'granted') {
-    throw new Error('Permission to access local fonts not granted.');
-  }
-} catch (err) {
-  // A `TypeError` indicates the 'local-fonts'
-  // permission is not yet implemented, so
-  // only `throw` if this is _not_ the problem.
-  if (err.name !== 'TypeError') {
-    throw err;
-  }
-}
-```
+{% Aside %} The Local Font Access API is currently only available on desktop versions of Chromium,
+but not on mobile operating systems like Android or iOS. {% endAside %}
 
 #### Enumerating local fonts
 
-Once the permission has been granted, you can then call `self.queryLocalFonts()`
-to ask the browser for the locally installed fonts, which it
-will display in a picker for the user to select all or a subset from to be shared with the page.
-This results in an array that you can loop over. Each font is represented as a `FontData` object
-with the properties `family` (for example, `"Comic Sans MS"`), `fullName` (for example,
-`"Comic Sans MS"`), `postscriptName` (for example, `"ComicSansMS"`), and `style` (for example, `"Regular"`).
+To obtain a list of the locally installed fonts, you need to call `window.queryLocalFonts()`. The
+first time, this will trigger a permission prompt, which the user can approve or deny. If the user
+approves their local fonts to be queried, the browser will display the list in a picker for the user
```

Comment:
That's not the case for me. A simple prompt shows up not a list in a picker anymore.

<img width="354" alt="image" src="https://user-images.githubusercontent.com/634478/176906077-309651ae-b640-4f06-b371-c6b2bdad9f89.png">